main.go

```diff
@@ -100,3 +115,66 @@ func renderPage(page string) string {
 	}
 	return result
 }
+
+type pageCache struct {
+	diskv *diskv.Diskv
+}
+
+// CachedPage is a page stored in the diskv cache.
+type CachedPage struct {
+	// The page content.
+	Page string
+	// Unix time (seconds).
+	CreatedAt int64
+}
+
+func (c *CachedPage) isValid(now int64) bool {
+	return now-c.CreatedAt < 60*60*24 // 1 day.
+}
+
+func newCache() *pageCache {
+	currentUser, err := user.Current()
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Could not read home directory: %v\n", err)
+		os.Exit(-1)
+	}
+
+	return &pageCache{
+		diskv: diskv.New(diskv.Options{
+			BasePath:     filepath.Join(currentUser.HomeDir, ".tldr", "cache"),
+			Transform:    func(key string) []string { return []string{} },
+			CacheSizeMax: 1024 * 1024,
+		})}
+}
+
+func (c *pageCache) write(command string, page string) {
```

Comment:
here and in read(), I think the parameter named "command" is technically correct, but it's so overloaded in programming that it may be misleading. I'd expect command to be named "page" (as the man man refers to the argument "page", and  the page "contents", but really just a siggestion